plugins/catalog/src/components/EntityPageLayout/EntityPageLayout.tsx

```diff
@@ -39,14 +48,14 @@ const EntityPageTitle = ({
   </Box>
 );
 
-function headerProps(
+const headerProps = (
   kind: string,
   namespace: string | undefined,
   name: string,
   entity: Entity | undefined,
-): { headerTitle: string; headerType: string } {
+): { headerTitle: string; headerType: string } => {
   return {
-    headerTitle: `${name}${
+    headerTitle: `${kind === 'user' ? toTitleCase(name) : name}${
```

Comment:
In order to reuse EntityPageLayout component I see no other way to do this. Without this check user profile will gave something like: name.surname in their title instead of Name Surname, and ideas how that can be done?